app/assets/v2/js/pages/bounty_details.js

```diff
@@ -223,22 +228,51 @@ var callbacks = {
     var tokens = val['tokens'];
     var decimals = 3;
 
-    var ui_elements = [];
+    let crowdfunded_tokens = [];
+    let tooltip_info = [];
+
+    crowdfunded_tokens.push({
+      amount: token_value_to_display(result['value_in_token'], decimals),
+      token: result['token_name']
+    });
 
+    tooltip_info.push('<p class="font-weight-bold m-0 pb-1" style="color:#775EC7;">Intial : ' +
+      $('#value_in_token').html() + '</p>');
+
+    if (usd_value) {
+      tooltip_info.push('<p class="m-0" style="margin-top: 3px;">Crowdfunding worth $' + usd_value + '</p>');
+      $('#value_in_usdt').html(result['value_in_usdt'] + usd_value);
+    }
+  
     for (var token in tokens) {
       if (token) {
         var val = tokens[token];
+        var funding = normalizeAmount(val, decimals);
 
-        ui_elements.push(Math.round(val * 10 ** decimals) / 10 ** decimals + ' ' + token);
+        if (crowdfunded_tokens.filter(fund => fund.token === token).length > 0) {
```

Comment:
no

Reply:
I kind of agree with @stickler-ci here. Though in this case it works perfectly fine, but in general this might often mess up the logic of the code, hence assignment and functions are not recommended inside an if condition or loop. So I would say maybe declare a variable.